lua/asllib.lua

```diff
@@ -4,6 +4,16 @@
 --
 Asllib = {} -- how should we refer to the below?
 
+-- clamp to bounds, must be greater than zero
+function boundgz( n )
+    if type(n) == 'function' then
+        return function()
+            nn = n()
```

Comment:
maybe make the `nn` declaration `local`?

Reply:
good catch!